inc/alert.class.php

```diff
@@ -216,6 +216,27 @@ public function rawSearchOptions()
         return $tab;
     }
 
+    public function post_updateItem($history = true)
+    {
+        // if close is not allowed update all user alerts to force display
+        if (
+            isset($this->input['is_close_allowed'])
+            && !$this->input['is_close_allowed']
+        ) {
+            $alert_user = new PluginNewsAlert_User();
+            //get all Alert_User for this alert where state is hidden
+            $all_alert = $alert_user->find(
+                [
+                    'plugin_news_alerts_id' => $this->getID(),
+                    'state' => PluginNewsAlert_User::HIDDEN
+                ]
+            );
+            foreach ($all_alert as $alert) {
+                //update state to 0 to force display
+                $alert_user->update(['id' => $alert['id'], 'state' => 0]);
```

Comment:
can you create a new constant for ```state = 0``` and use it when necessary

```PluginNewsAlert_User::VISIBLE```

Reply:
[3930d98](https://github.com/pluginsGLPI/news/pull/143/commits/3930d98366c80b5180dcd54140905b3c0941474f) :+1: